attachments_to_filesystem/models/ir_attachment.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import pytz
+import logging
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from openerp.osv.orm import Model
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class IrAttachment(Model):
+    _inherit = 'ir.attachment'
+
+    def _attachments_to_filesystem_init(self, cr, uid, context=None):
+        """Set up config parameter and cron job"""
+        module_name = __name__.split('.')[-3]
+        ir_model_data = self.pool['ir.model.data']
+        ir_cron = self.pool['ir.cron']
+        location = self.pool['ir.config_parameter'].get_param(
+            cr, uid, 'ir_attachment.location')
+        if location:
+            # we assume the user knows what she's doing. Might be file:, but
+            # also whatever other scheme shouldn't matter. We want to bring
+            # data from the database to there
+            pass
+        else:
+            ir_model_data._update(
+                cr, uid, 'ir.config_parameter', module_name,
```

Comment:
Please set noupdate to True in this call to prevent this key to be removed upon module uninstallation, which would make all attachments unavailable.